packages/host/app/components/operator-mode/profile-info-popover.gts

```diff
@@ -31,59 +36,187 @@ export default class ProfileInfoPopover extends Component<ProfileInfoPopoverSign
+        justify-content: space-between;
+        align-items: center;
+        flex-wrap: wrap;
+        gap: var(--boxel-sp-lg);
+        margin-bottom: auto;
+        padding-top: var(--boxel-sp-lg);
+        border-top: 1px solid var(--boxel-dark);
+      }
+      .info-group {
+        display: flex;
+        flex-direction: column;
+        gap: var(--boxel-sp-xxs);
+      }
+      .label {
         color: var(--boxel-dark);
-        text-transform: uppercase;
+        font: var(--boxel-font-xs);
+      }
+      .out-of-credit-info {
+        font: var(--boxel-font-sm);
+      }
+      .info-group .value.out-of-credit {
+        color: #ff3838;
+      }
+      .info-group .value {
+        color: var(--boxel-dark);
+        font: 700 var(--boxel-font-sm);
+        font-style: capitalize;
       }
     </style>
 
-    <div class='profile-popover' data-test-profile-popover ...attributes>
+    <div
+      class={{cn 'profile-popover' display-choose-plan-button=this.isFreePlan}}
+      data-test-profile-popover
+      ...attributes
+    >
       <header class='header'>
-        <div class='label'>
-          Signed in as
-        </div>
-
         <BoxelButton
           @kind='secondary-light'
-          @size='extra-small'
+          @size='small'
           {{on 'click' @toggleProfileSettings}}
           data-test-settings-button
         >
           Settings
         </BoxelButton>
+
+        <BoxelButton
+          @kind='primary-dark'
+          @size='small'
+          {{on 'click' this.logout}}
+          data-test-signout-button
+        >
+          Sign Out
+        </BoxelButton>
       </header>
 
       <ProfileInfo />
-
-      <BoxelButton
-        {{on 'click' this.logout}}
-        @kind='primary-dark'
-        data-test-signout-button
-      >
-        Sign out
-      </BoxelButton>
+      <div class='credit-info' data-test-credit-info>
+        <div class='info-group'>
+          <span class='label'>Membership Tier</span>
+          <span class='value' data-test-membership-tier>{{this.plan}}</span>
+        </div>
+        {{#if this.isChangePlanDisplayed}}
+          <BoxelButton
+            @kind='secondary-light'
+            @size='small'
+            data-test-change-plan-button
+            {{on 'click' @toggleProfileSettings}}
+          >Change plan</BoxelButton>
+        {{/if}}
+        <div class='info-group'>
+          <span class='label'>Monthly Credit</span>
+          <span
+            class={{cn 'value' out-of-credit=this.isOutOfCredit}}
+            data-test-monthly-credit
+          >{{this.monthlyCredit}}</span>
+        </div>
+        {{#if this.isOutOfCredit}}
+          <span class='out-of-credit-info' data-test-out-of-credit>You have used
+            all of credits from the free trial, please upgrade your plan.</span>
+        {{/if}}
+        <div class='info-group'>
+          <span class='label'>Additional Credit</span>
+          <span class='value' data-test-additional-balance>{{if
```

Comment:
I think we should consider always displaying the remaining extra credits. Imagine a situation where user can downgrade from Creator to Free plan, and they have some extra credits left. We need to display that amount somewhere.